code/datums/shuttles/_shuttle.dm

```diff
@@ -54,6 +54,7 @@
 
 /datum/map_template/shuttle/load(turf/T, centered, register=TRUE)
 	. = ..()
+	sleep(1 SECONDS) // Ugly temporary hack fix to make sure modular shuttles have time to assemble themselves before being moved.
```

Comment:
Ideally (sort of), we should delay loading only for modular shuttles, not every one.